### PR TITLE
PMM-11330 Change for new integration setup pdpgsql

### PR DESCRIPTION
**Pull request by peterSirotnak**
https://jira.percona.com/browse/PMM-0

- [ ] Links to relevant pull requests

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3048-034f592
client docker - perconalab/pmm-client-fb:PR-3048-034f592
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3048-034f592.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3048-034f592&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3048-034f592.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3048-d79bcb0
client docker - perconalab/pmm-client-fb:PR-3048-d79bcb0
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3048-d79bcb0.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3048-d79bcb0&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3048-d79bcb0.tar.gz

File: .github/workflows/pmm2-testsuite.yml [hunk 1]
```diff
@@ -155,16 +155,25 @@ jobs:
         run: sudo bats ./pmm-tests/pmm-2-0-bats-tests/ps-specific-tests.bats
 
       - name: Run Setup for pdpgsql 13
+        working-directory: pmm-integration
         if: ${{ matrix.db-type == 'pdpgsql13' }}
-        run: sudo -E bash -x ./pmm-tests/pmm-framework.sh --addclient=pdpgsql,1 --pdpgsql-version=13 --pmm2
+        run: |
+          sudo npm install
```
Comment (yurkovychv) [inline .github/workflows/pmm2-testsuite.yml]: please move the `install` commands to a separate step in the beginning. And use `sudo npm ci`

**Review by yurkovychv — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3048-92a712b
client docker - perconalab/pmm-client-fb:PR-3048-92a712b
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3048-92a712b.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3048-92a712b&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3048-92a712b.tar.gz

File: .github/workflows/pmm2-testsuite.yml (hunk 1 above)
Comment (peterSirotnak) [reply .github/workflows/pmm2-testsuite.yml]: Moved.

**Review by yurkovychv — APPROVED** [review]

**Pull request merged**